### DISCUSSION TITLE
new issue templates causes issues without titles

### DISCUSSION THREAD

**Discussion by Luap99** [containers/podman#17431] [General]
Normally github will not allow you to submit an issue without title, however because the new template will automatically prefix the title users can submit it with just that.
This happened multiple times in the last few week so I think we should do something about it. 
It is rather annoying to see issues without titles because you have to open them to see what they are about.

I think a simple solution is not prefix the titles, we already apply the correct label anyway and the prefix is just unnecessary noise to me.

**Comment by Luap99** [comment]
@containers/podman-maintainers WDYT?

**Reply by vrothberg** [reply]
+1
I've seen this issue as well and think your proposal will fix it.

**Comment by edsantiago** [comment]
To me, subject lines are precious. `[Bug]` and `[Feature]` are wastes of valuable real estate.

**Reply by Luap99** [reply]
Agreed, PR to remove it: https://github.com/containers/podman/pull/17437
